src/pages/Admin/Admin_v2.tsx

```diff
@@ -139,9 +138,11 @@ const Admin_v2: React.FC = () => {
     }
   };
 
+  const hasSelectedVenue = !!selectedVenue;
+
   return (
-    <>
-      <S.Wrapper hasSelectedVenue={!!selectedVenue}>
+    <WithNavigationBar>
```

Comment:
Should this have the `fullscreen` prop set on it?

Reply:
`fullscreen` adds overflow for `x` and `y` which adds extra scrolls which are unnecessary.